src/modules/registrypreview/RegistryPreviewUILib/RegistryPreviewMainPage.xaml

```diff
@@ -274,11 +275,19 @@
                                             <MenuFlyoutItem x:Uid="ContextMenu_CopyValueWithPath" Command="{Binding CopyToClipboardWithPath_Click}" />
                                         </MenuFlyout>
                                     </StackPanel.ContextFlyout>
+                                    <Button
+                                        Padding="2"
+                                        Click="ButtonEnhancePreview_Click"
+                                        Content="&#xe71e;"
+                                        FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                        Visibility="{Binding ShowPreviewButton, Mode=OneWay}">
```

Comment:
@niels9001 
I tried to use "BoolToVisibility" converter here. But it crashes the app and intellisense doesn't suggest it. What do I do wrong?